### PR TITLE
Ensure MongoRDD.toDF uses the DefaultSource

### PR DESCRIPTION
**Pull request by rozza**
That way any further filtering of the DataFrame can be pushed into
the aggregation framework.

SPARK-80

### CODE REVIEW

File: src/test/scala/com/mongodb/spark/sql/MongoDataFrameSpec.scala [hunk 1]
```diff
@@ -135,6 +135,17 @@ class MongoDataFrameSpec extends RequiresMongoDB {
     df.schema should equal(createStructType(expectedSchema.fields.filter(p => p.name != "name")))
   }
 
+  it should "use any pipelines when set vai the MongoRDD" in withSparkContext() { sc =>
```
Comment (evanchooly) [inline src/test/scala/com/mongodb/spark/sql/MongoDataFrameSpec.scala]: s/vai/via/

**Comment by rozza** [comment]
Good spot @evanchooly - updated.

**Comment by jyemin** [comment]
LGTM

**Comment by evanchooly** [comment]
lgtm

**Pull request closed without merging**